src/main/java/gameframework/motion/overlapping/OverlapRulesApplierDefaultImpl.java

```diff
@@ -18,28 +18,32 @@ public void applyOverlapRules(Vector<Overlap> overlaps) {
 		}
 	}
 
-	protected void applySpecificOverlapRule(Overlappable e1, Overlappable e2) {
+	/**
+	 * @param tryToReverseParameters
```

Comment:
Please add to the documentation that the method is commutative and that is the reason for this parameter.